src/agent/onefuzz-task/src/tasks/report/crash_report.rs

```diff
@@ -168,6 +168,14 @@ impl CrashTestResult {
                 // Use SHA-256 of call stack as dedupe key.
                 if let Some(unique_reports) = unique_reports {
                     let name = report.unique_blob_name();
+                    if let Some(jr_client) = jr_client {
```

Comment:
let's move this logic outside of the `unique_report` container check.
and rename to `crash_reported`